gradle-recommended-product-dependencies/src/main/java/com/palantir/gradle/dist/RecommendedProductDependenciesPlugin.java

```diff
@@ -25,9 +25,6 @@
 
 public class RecommendedProductDependenciesPlugin implements Plugin<Project> {
 
-    public static final String RESOURCE_PATH =
-            RecommendedProductDependencies.SLS_RECOMMENDED_PRODUCT_DEPS_KEY + "/product-dependencies.json";
```

Comment:
This is used in at least this test in another repo: https://github.com/palantir/gradle-conjure/blob/947daccbc62a144195cca972a831882654aff10a/gradle-conjure/src/test/groovy/com/palantir/gradle/conjure/ConjureJavaLocalCodegenPluginIntegrationSpec.groovy#L211. I'd probably just keep it where it is to avoid any breaks.